bgpd/bgp_evpn_vty.c

```diff
@@ -2884,7 +2884,7 @@ DEFUN (no_bgp_evpn_advertise_type5,
 		if (CHECK_FLAG(bgp_vrf->af_flags[AFI_L2VPN][SAFI_EVPN],
 			       BGP_L2VPN_EVPN_ADVERTISE_IPV6_UNICAST)) {
 			bgp_evpn_withdraw_type5_routes(bgp_vrf, afi, safi);
-			UNSET_FLAG(bgp_vrf->vrf_flags,
+			UNSET_FLAG(bgp_vrf->af_flags[AFI_L2VPN][SAFI_EVPN],
 				   BGP_L2VPN_EVPN_ADVERTISE_IPV6_UNICAST);
```

Comment:
just to be sure. touching this flag does not require to flush some other contexts ?

Reply:
The code to do that is already there, we were just unsetting the wrong flag for when we were clearing it.

Reply:
ok